ui/tests/integration/components/replication-secondary-card-test.js

```diff
@@ -0,0 +1,94 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+
+const TITLE = 'States';
+const HAS_ERROR_CLASS = false;
+
+const REPLICATION_DETAILS = {
+  state: 'stream-wals',
+  connection_state: 'ready',
+  lastWAL: 0,
+  lastRemoteWAL: 10,
+  delta: 10,
+};
+
+const STATE_ERROR = {
+  state: 'idle',
+  connection_state: 'ready',
+  lastWAL: 0,
+  lastRemoteWAL: 10,
+  delta: 10,
+};
+
+const CONNECTION_ERROR = {
+  state: 'stream-wals',
+  connection_state: 'transient_failure',
+  lastWAL: 0,
+  lastRemoteWAL: 10,
+  delta: 10,
+};
+
+module('Integration | Enterprise | Component | replication-secondary-card', function(hooks) {
+  setupRenderingTest(hooks);
+
+  hooks.beforeEach(function() {
+    this.set('replicationDetails', REPLICATION_DETAILS);
+    this.set('hasErrorClass', HAS_ERROR_CLASS);
+    this.set('title', TITLE);
+  });
+
+  test('it renders', async function(assert) {
+    await render(
+      hbs`<ReplicationSecondaryCard @replicationDetails={{replicationDetails}} @title={{title}} @hasErrorClass={{hasErrorClass}}/>`
```

Comment:
out of curiosity, is it necessary to manually pass in `hasErrorClass`? isn't it computed as false by default?

Reply:
Good catch.  I did not have the `hasErrorClass` set as false by default on the component js file.  I amended and removed where appropriate from the tests.